test/k8s-integration/driver-config_test.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "testing"
+
+// WARNING: Not an official test - run this as a quick n dirty way to test your
+// template generation code :)
+func TestGenerateDriverConfigFile(t *testing.T) {
+	_, err := generateDriverConfigFile("/usr/local/google/home/dyzz/go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver", "sc-standard.yaml", "gke")
```

Comment:
Did you want to commit this file as-is?

I am in favor of a test that renders the template and then verifies it parses as valid yaml...

Reply:
I'm not convinced we need a test for the test. This was just for my personal iteration speed. I would rather remove this file than spend more time on it

Reply:
:+1: